third_party/docfx-doclet-143274/src/test/java/com/microsoft/util/ApiVersionTest.java

```diff
@@ -70,4 +73,37 @@ public void equals() {
   private ApiVersion parse(String s) {
     return ApiVersion.parse(s).orElseThrow(() -> new IllegalStateException("Unable to parse " + s));
   }
+
+  @Test
+  public void testRecommendation_PrioritizesReleaseVersions() {
+    ImmutableList<ApiVersion> versions =
+        ImmutableList.of(parse("v101beta"), parse("v2p1"), parse("v1p14alpha15"));
```

Comment:
Consider adding a test case where there are 2 release versions for completeness (maybe 2 release versions and one prerelease or something).

Reply:
Done